tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_device_RTL.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I got pixels difference with this test:
TEST-UNEXPECTED-ERROR | test_settings_device_RTL.py TestSettingsRTL.test_settings_app | AssertionError: WARNING: 1376 pixels mismatched between screenshots/test_settings_app_developer_flame_ar_9+2015-08-05-18-55-51.png and reference_images/test_settings_app_developer_flame_ar_9.png